bitnami/milvus/README.md

```diff
@@ -1592,6 +1600,16 @@ To enable Ingress integration, set `attu.ingress.enabled` to `true`. The `attu.i
 
 The chart also facilitates the creation of TLS secrets for use with the Ingress controller, with different options for certificate management. [Learn more about TLS secrets](https://docs.bitnami.com/kubernetes/apps/mastodon/administration/enable-tls-ingress/).
 
+## Upgrading
+
+### To 2.0.0
+
+This major updates the Kafka subchart to its newest major, 24.0.0. This new version refactors the Kafka chart architecture and requires manual actions during the upgrade. For more information on this subchart's major, please refer to [Kafka upgrade notes](https://github.com/bitnami/charts/tree/main/bitnami/kafka#to-2400).
```

Comment:
Could you add that now use "User for SASL authentication"? I think that information is useful for the users.

Reply:
Updated the Upgrading notes with the modified SASL parameters.